### DISCUSSION TITLE
Unable to display custom link on device table

### DISCUSSION THREAD

**Discussion by travisjohnsonga** [nautobot/nautobot#4549] [General]
Hello all - 

Evaluating Nautobot vs Netbox.

One thing I have created in Netbox is custom links on the device objects to link to ssh:// which I can use to launch putty.  

I'm not able to add links as columns in nautobot via the configure table button.

Is there another way to do this, or is it missing from nautobot?

**Comment by itdependsnetworks** [comment]
It's the last item on "extensibility", sometimes it is a bit hidden if there are too many options, such as on demo.nautobot.com, but just scroll down.

**Reply by travisjohnsonga** [reply]
I can create the custom link - however, it cannot be displayed in the device table view, it only shows in the device detail view.

**Reply by travisjohnsonga** [reply]
screenshot below is from netbox

**Comment by travisjohnsonga** [comment]
![custom link in table](https://github.com/nautobot/nautobot/assets/108543973/93043d4b-b96c-474f-b48d-bbbf8fe8cfbc)

**Reply by jvanderaa** [reply]
So this view has brought the custom links out of the Device Detail into the Device List view. Shouldn't be too difficult to get into Nautobot from that side of things would be my thought.

**Reply by jvanderaa** [reply]
It will likely start with a FR. @lampwins thoughts?

**Reply by travisjohnsonga** [reply]
Thank you - it's a very useful feature.  My team has started using these links rather than trying to keep up with securecrt databases to pass around.  Having them on the table view allows for quickly connecting to multiple devices without having to go to each device page.

**Reply by travisjohnsonga** [reply]
just to bump this - do I need to put in a FR for this, or was one started from this thread already for it?
